src/parsing/musicxml/annotation.ts

```diff
@@ -21,6 +21,15 @@ export class Annotation {
     return new Annotation(config, log, machine.getText(), 'center', 'bottom');
   }
 
+  static fromFingering(config: Config, log: Logger, musicXML: { fingering: musicxml.Fingering }): Annotation | undefined {
+    const number = musicXML.fingering.getNumber();
+    if (!number) {
```

Comment:
I don't know if finger 0 is valid (I guess semantics are up to the transcriber), but would you make this check slightly more robust checking presence with `typeof number !== 'number'`?